Version 1.3.0

guillaumecourt opened the pull request:
Correction et évolution : 
- Prise en charge du multilinguisme par le module
- Renvoi des paniers en erreur
- contrôle sur l'UA et la clé CYD
- ajout du numéro de version du module à l'app
- modification de l'onglet Google Analytics
- suppression des espaces lors de la saisie
- mise à jour des bannières

thribiere committed:
fix #240 : add validation rules for google UA and checkyourdata Token

thribiere committed:
fix #244 : hide checkbox tracker activation (default activated)
+ adjust translation

thribiere committed:
fix #244 : add infos sent to app

thribiere committed:
fix #257 : add analytics ec module require

thribiere committed:
preparation de la v1.2.7

Commit:
Correction Récupération GCID par le navigateur du client si le GCID n'est pas trouvé par le serveur

Commit:
Merge GCID

# Conflicts:
#	checkyourdata.php

Commit:
Correction coding standards

Commit:
fix #261, fix #244

#261: Onglet Google Analytics
#244: Onglet Google Analytics

Commit:
Fix envoi commandes et paniers bloqués

Fix envoi commandes et paniers bloqués
ajout timeout pour l'appel à l'APP
refactoring code (envoi commandes et paniers bloqués)

Mekki78 commented:
Hi,

Could you correct this following note:
- Use version_compare if you want to make a check between two PrestaShop version 
- Use only English comment (you have English and French comments in your code)
- Add index.php in /views/img (security)

Your configuration page in your Back Office is made also with English and French text on the same page. 

![download 6](https://cloud.githubusercontent.com/assets/6247743/12555154/0bc220fa-c380-11e5-9982-12d4abe13cbc.png)

PrestaShop documentation for module translation:
http://doc.prestashop.com/display/PS16/Module+translation

Thank you in advance.

Commit:
Retour pull requests Version 1.2.7

Use version_compare
Use only English comment
Add index.php in /views/img

guillaumecourt commented:
Hi,

I corrected : 
 - version_compare
 - comments 
 - add index.php in /views/img
 - configuration page in Back Office

Thank you in advance.

Commit:
Changement processus de création de compte

Modification du processus de création de compte check your data en utilisant l'App

Commit:
Changement processus de création de compte

Adaptation version 1.4 et 1.5

guillaumecourt commented:
Bonjour @Mekki78, @Quetzacoalt91  

Je me permet de passer en français pour enrichir mon vocabulaire et ainsi vous informer que nous souhaitons déployer une nouvelle version.

Cette "Pull Request" datant de plus de 3 semaines et étant peu habitué sur Git, je voudrais savoir comment annuler cette "Pull Request" afin de pousser une version 1.3.

Merci d'avance

Quetzacoalt91 commented:
Bonjour @guillaumecourt,

La seule chose à faire est de rajouter les commits sur la branche master. La PR se mettra à jour d'elle même. Ensuite, l'intitulé de la PR peut être mise à jour en cliquant sur le bouton "Edit" situé à côté.

Cordialement,

Commit:
Validation Presta

Validation Presta

Commit:
Correction image admin

déplacement de l'image de fond du back-office

Commit:
Correction currency, validation, cache

Correction de l'envoi de la currency
Correction validation
Vidage du cache à l'install

Commit:
Ajout periode d'essai + correction affichage msg

Ajout periode d'essai à partir de l'APP Correction affichage msg

Commit:
Traduction + style + info autre tracker

Modification des textes
Modification taille panel
Ajout popup (PS> 1.6) pour les autres analytics

Quetzacoalt91 commented:
Bonjour @guillaumecourt,

Suite aux précédents messages, que devons nous faire de cette pull-request ?

Cordialement

guillaumecourt commented:
Bonjour @Quetzacoalt91,

Nous finalisons la version et on pousse demain.

Cordialement.

Commit:
Correction css

Commit:
Correction configuration

Commit:
Merge branch '1.3'

Commit:
Correction Traduction

guillaumecourt commented:
Bonjour @Quetzacoalt91, @mmoustai 

La version 1.3.0 est poussée. Cette version tient compte de votre précédent retour de correction à effectuer.

Auriez vous une visibilité concernant le déploiement sur votre Master ?

Bien cordialement.
Guillaume Court

guillaumecourt commented:
Bonjour @Quetzacoalt91, @mmoustai 
Cela fait 14 jours que les sources ont été poussées, auriez vous une visibilité sur la validation de la Pull Requests ou de son avancement.

Cordialement.

Commit:
Modification logo

Commit:
Optimisation enregistrement par token

Récupération automatique des informations du compte

Commit:
Modification descriptif

Modification descriptif et logo

Commit:
Modification descriptif

Modification descriptif
+ optimisation sauvegarde configuration

Commit:
Modification descriptif

Modification descriptif
